cmd/openshift-diagnostics/main.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"math/rand"
+	"os"
+	"runtime"
+	"time"
+
+	"k8s.io/apiserver/pkg/util/logs"
+
+	"github.com/openshift/origin/pkg/cmd/util/serviceability"
+	"github.com/openshift/origin/pkg/oc/admin/diagnostics"
+	"github.com/spf13/cobra"
+
+	// install all APIs
+	_ "github.com/openshift/origin/pkg/api/install"
+	_ "k8s.io/kubernetes/pkg/api/install"
+	_ "k8s.io/kubernetes/pkg/apis/autoscaling/install"
+	_ "k8s.io/kubernetes/pkg/apis/batch/install"
+	_ "k8s.io/kubernetes/pkg/apis/extensions/install"
```

Comment:
do we need apps ?

Reply:
> cmd/openshift-diagnostics/main.go

No, the main one installs all.  We are in need of an overhaul for installation though.